GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -291,6 +292,10 @@
     private boolean lastCollection = false;
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+    private Map<Long,List<QuestionOptionDto>> optionMap = new HashMap<Long,List<QuestionOptionDto>>();
+    private Map<Long,Boolean> allowOtherMap = new HashMap<Long,Boolean>();
+    private Map<String,Integer> optionCache = new HashMap<String,Integer>();
```

Comment:
May I suggest we rename this to `optionsPositionCache`  while reading through I was wondering what exactly we are caching as it was a bit confusing with the name